### PR TITLE
Suraj kumar haiku

### PR DESCRIPTION
**Pull request by Surajkumar98012**
thanks for reading my pull request

### CODE REVIEW

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Thank you for submitting this haiku! There are some changes that need to be made to the syllable count in order to have a valid haiku.

A haiku is composed of 3 lines. The first line is 5 syllables long, the second line is 7 syllables long, and the third line is 5 syllables long. Although the majority of the haikus here are written in English, you can submit a haiku in any human language.

Please consult our [contributing guidelines](https://github.com/do-community/cloud_haiku#contributing) if you need extra guidance. Once you amend this pull request, we can merge it in.

*Please note: after more than two days of inactivity, pull requests are subject to being closed and marked invalid by maintainers.*

**Comment by Surajkumar98012** [comment]
Ok I will edit it today

File: _haikus/2022_opensource.md
Comment (ltagliaferri) [inline _haikus/2022_opensource.md:7]:
```suggestion
i'm a beginner<br>
```

File: _haikus/2022_opensource.md
Comment (ltagliaferri) [inline _haikus/2022_opensource.md:8]:
```suggestion
in the field of open source<br>
```

File: _haikus/2022_opensource.md
Comment (ltagliaferri) [inline _haikus/2022_opensource.md:9]:
```suggestion
glad to contribute<br>
```

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Suggesting these changes for the syllables to be correct

**Review by ltagliaferri — APPROVED** [review]
Thank you for this haiku and for contributing to open source 🎉

**Pull request merged**
